src/test/HelmetTest.js

```diff
@@ -1161,6 +1189,24 @@ describe("Helmet", () => {
                     .to.be.a("string")
                     .that.equals(`<script src="foo.js" async="" ${HELMET_ATTRIBUTE}="true"></script>`);
             });
+
+            it("won't render tag when primary attribute is null", () => {
+                ReactDOM.render(
+                    <Helmet
+                        script={[
+                            {
+                                src: undefined,
```

Comment:
To your point, yes, this would not render a tag:
```
{
    innerHTML: "function test() {console.log('test');} test();",
    src: undefined
}
```
because it found a primary key without a value, BUT you should really never be specifying more than one primary key for your tag.

This, on the other hand, will render a tag because the for loop only uses one primary key, and takes the last one it finds:

```
{
    src: undefined,
    innerHTML: "function test() {console.log('test');} test();"
}
```

But again, this use case would be misuse.